tests/test_nautilus_librarian/test_typer/test_commands/test_workflows/test_actions/test_message.py

```diff
@@ -0,0 +1,11 @@
+from nautilus_librarian.typer.commands.workflows.actions.action_result import Message
+
+
+def test_messages_can_be_compared():
+    msg1 = Message("message 1")
+    msg2 = Message("message 1")
+
```

Comment:
I would remove this new line between msg2 and msg3 declarations

Reply:
Fixed